pack/return/rttcu_encounter.json

```diff
@@ -20,5 +539,569 @@
+        "faction_code": "mythos",
+        "illustrator": "Stephen Somers",
+        "name": "Nightmare Breach",
+        "pack_code": "rttcu",
+        "position": 58,
+        "quantity": 1,
+        "shroud": 5,
+        "text": "<b>Cosmos</b> - Connect in a direction of your choice, but only onto a spot where an empty space exists. Remove that empty space from the game instead of shuffling it back into its owner's deck. Then, discard each other copy of that card from play and from each investigator's hand.",
+        "traits": "Otherworld. Void.",
+        "type_code": "location"
+    },
+    {
+        "back_illustrator": "Uwe Jarling",
+        "back_name": "Cosmos",
+        "back_flavor": "The stars danced mockingly, almost shifting now and then to form pale signs of doom that one might wonder one had not seen and feared before; and ever the winds of aether howled of vague blackness and loneliness beyond the cosmos.\n- H. P. Lovecraft, <u>The Dream-Quest of Unknown Kadath</u>",
+        "back_traits": "Otherworld.",
+        "clues": 0,
+        "code": "54059",
+        "double_sided": true,
+        "encounter_code": "return_to_before_the_black_throne",
+        "encounter_position": 4,
+        "faction_code": "mythos",
+        "illustrator": "Greg Bobrowski",
+        "name": "Interstellar Abyss",
+        "pack_code": "rttcu",
+        "position": 59,
+        "quantity": 1,
+        "shroud": 6,
+        "text": "<b>Cosmos</b> - Connect in the direction nearest to The Black Throne (or any direction of The Black Throne is not in play).\nWhile you are at Interstellar Abyss, you cannot spend clues.",
+        "traits": "Otherworld. Void.",
+        "type_code": "location"
+    },
+    {
+        "back_illustrator": "Uwe Jarling",
+        "back_name": "Cosmos",
+        "back_flavor": "The stars danced mockingly, almost shifting now and then to form pale signs of doom that one might wonder one had not seen and feared before; and ever the winds of aether howled of vague blackness and loneliness beyond the cosmos.\n- H. P. Lovecraft, <u>The Dream-Quest of Unknown Kadath</u>",
+        "back_traits": "Otherworld.",
+        "clues": 2,
+        "clues_fixed": true,
+        "code": "54060",
+        "double_sided": true,
+        "encounter_code": "return_to_before_the_black_throne",
+        "encounter_position": 5,
+        "faction_code": "mythos",
+        "illustrator": "Greg Bobrowski",
+        "name": "Winding Gulf",
+        "pack_code": "rttcu",
+        "position": 60,
+        "quantity": 1,
+        "shroud": 2,
+        "text": "<b>Cosmos</b> - Connect to the leftmost revealed location in a direction of your choice. Then, you must either immediately move to Winding Gulf or place 1 doom on Azathoth.\n<b>Haunted</b> - Take 1 damage and 1 horror.",
+        "traits": "Otherworld. Void.",
+        "type_code": "location"
+    },
+    {
+        "code": "54061",
+        "cost": null,
+        "deck_limit": 1,
```

Comment:
Remove this line, as it lets players add the Steed to their decks.

Reply:
Pity!